projects/kit/tokens/countries-masks.ts

```diff
@@ -217,4 +218,12 @@ export const TUI_COUNTRIES_MASKS = tuiCreateToken<Record<TuiCountryIsoCode, stri
     [TuiCountryIsoCode.ZA]: '+27##-###-####',
     [TuiCountryIsoCode.ZM]: '+260##-###-####',
     [TuiCountryIsoCode.ZW]: '+263#-######',
-});
+};
+
+export const TUI_COUNTRIES_MASKS = tuiCreateToken(TUI_COUNTRIES_DEFAULT_MASKS);
```

Comment:
@waterplea @vladimirpotekhin 

We have plans to use `@maskito/phone` in `v4.0`.
Do we actually require this new token in `main`-branch ? (probably, it will be never released 🤔 )

Reply:
yes, in fact, we need these changes in the 3.x branch only

Reply:
@nguyenphutrong could you please make these changes into `v3.x` branch instead of `main` so it will be available for everybody in the current major version. Upcoming 4.0 will have this mechanism reworked.